data/group.go

```diff
@@ -50,17 +50,17 @@ func (g *group) seal(id uuid.UUID, cryptor crypto.CryptorInterface) (SealedGroup
 	return SealedGroup{id, ciphertext, wrappedKey}, nil
 }
 
-// unseal decrypts the sealed group.
-func (g *SealedGroup) unseal(cryptor crypto.CryptorInterface) (group, error) {
-	plainGroup := group{}
+// Unseal decrypts the sealed group.
+func (g *SealedGroup) Unseal(cryptor crypto.CryptorInterface) (Group, error) {
+	plainGroup := Group{}
 	if err := cryptor.Decrypt(&plainGroup, g.ID.Bytes(), g.WrappedKey, g.Ciphertext); err != nil {
-		return group{}, err
+		return Group{}, err
 	}
 	return plainGroup, nil
 }
 
-// verify checks the integrity of the sealed group.
-func (g *SealedGroup) verify(cryptor crypto.CryptorInterface) bool {
-	_, err := g.unseal(cryptor)
+// Verify checks the integrity of the sealed group.
```

Comment:
Is there a reason why SealedGroup and SealedUser still have a verify method, while all other Sealed-something do not? (There probably is, I just wanted to mention it).

Reply:
Yes, they are used [here](https://github.com/cyber-crypt-com/encryptonize-lib/blob/1a1ef47a6a5ae043a52b0f644db69ebb42f79a52/encryptonize.go#L100) and [here](https://github.com/cyber-crypt-com/encryptonize-lib/blob/1a1ef47a6a5ae043a52b0f644db69ebb42f79a52/utility.go#L68).